### DISCUSSION TITLE
Does frigate-notify work with frigate v0.14x?

### DISCUSSION THREAD

**Discussion by gniting** [0x2142/frigate-notify#71] [General]
If I upgrade my frigate installation to the latest 0.14.0 Beta 2, should I expect frigate-notify to be compatible with it?

**Comment by 0x2142** [comment]
Hi @gniting - So far from my initial testing, I haven't had any issues running it with the 0.14.0 betas. That being said, I've only tested with my development Frigate box & haven't upgraded my primary one yet. 

I don't expect any issues - but if you do try it out, please let us know if you run into anything!

**Reply by gniting** [reply]
Will do!

**Reply by gniting** [reply]
@0x2142 I've updated to frigate 0.14beta2 and v0.28 of frigate-notify worked with it without any hitch. 

Additionally, the latest frigate-notify update (v.0.3.0) also works perfectly.

**Reply by 0x2142** [reply]
Awesome! I appreciate the info! Thanks so much for reporting back 😊
